src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/SqlConnection.cs

```diff
@@ -204,10 +204,12 @@ public Stream ReadBlob(string dataTableName, string dataColumnName, long rowId)
             return stream;
         }
 
-        private Stream ReadBlob_InTransaction(string tableName, string columnName, long rowId)
+
+        private static byte[] mainPtr = Util.ToUtf8("main");
```

Comment:
readonly? Yeah, the array is still mutable but it's a small protection.

Reply:
Sure thing!